fix Gutenberg block rules with JSON path (WP-745) (#420)
Smartling/wordpress-localization-plugin: readme.txt

3. Track translation status within WordPress from the Submissions Board. View overall progress of submitted translation requests as well as resend updated content.

== Changelog ==
= 2.16.1 =
* Fixed Gutenberg block rules with JSON paths in nested Gutenberg blocks

= 2.16.0 =
* Removed ACF and manual relation handling obsolete expert settings. The behaviours controlled by these options have been reverted to the default values

= 2.15.1 =

> * Added options to prevent plugin from escaping post content and metadata before saving. It is useful if the installation has other plugins that hook into the post content or metadata saving and add extra escaping slashes to them

Reviewer: Since you touched, here is mine version :)
```suggestion
* Added options to prevent plugin from escaping post content and metadata before saving. This is useful if there are other plugins that hook on the "post content" or "save metadata" and add extra escaping slashes to them
```
